support-models/src/main/scala/com/gu/support/zuora/api/SubscribeRequest.scala

```diff
@@ -25,3 +28,14 @@ object SubscribeRequest {
 //The subscribe request documented here: https://www.zuora.com/developer/api-reference/#operation/Action_POSTsubscribe
 //fields are upper case to match the expected json structure
 case class SubscribeRequest(subscribes: List[SubscribeItem])
+
+import com.gu.support.encoding.Codec._
+
+object UpdateRedemptionDataRequest {
+  implicit val encoder: Encoder[UpdateRedemptionDataRequest] = deriveEncoder[UpdateRedemptionDataRequest].mapJsonObject(
+    _.renameField("gifteeIdentityId", "GifteeIdentityId__c")
+  )
+}
+
+case class UpdateRedemptionDataRequest(gifteeIdentityId: String, currentTerm: Int, currentTermPeriodType: PeriodType)
```

Comment:
we need to add the CreatedInRequestId__c field to this and update it.

if you have the patience, could be worth renaming the field to something like ProcessedInRequestId__c (create a new one in zuora, and delete the old one) as they are only needed while a step function exection is in progress AND zuora is having issues.

Reply:
🤦  Yes! I've updated the tests to catch this properly now as well